pkgs/applications/networking/feedreaders/feedreader/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchFromGitHub, cmake, curl
+, gettext, glib, gnome3, gst_all_1, json_glib
+, libaccounts-glib, libnotify, libsecret
+, libxmlxx, makeWrapper, pkgconfig, sqlite
+}:
+
+stdenv.mkDerivation rec {
+
+  name = pname + "-" + version;
+  pname = "FeedReader";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "jangernert";
+    repo = pname;
+    rev = "v" + version;
+    sha256 = "1jz4jlda1a7nfa2wjizmivh6bpdrprnhnxdcgj7wip71888rzbmy";
+  };
+
+  buildInputs = [
+    cmake curl gettext glib json_glib libaccounts-glib libnotify libsecret
+    libxmlxx makeWrapper pkgconfig sqlite
+  ] ++
+   (with gnome3; [
+     gnome_online_accounts gtk libgee libpeas libsoup rest webkitgtk vala
+   ]) ++
+   (with gst_all_1; [
+     gstreamer gstreamermm
+   ]);
+
+  buildPhase = ''
+    mkdir $out
+    cmake \
+      -DCMAKE_INSTALL_PREFIX=$out \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      .
+    make
+  '';
+
+  preFixup = ''
+    glib-compile-schemas $out/share/gsettings-schemas/${name}/glib-2.0/schemas
+    for exe in $out/bin/*
+    do
+      wrapProgram "$exe" \
```

Comment:
Add `wrapGAppsHook` to `nativeBuildInputs` to populate the variables automatically.